templates/items/parts/item-activation.hbs

```diff
@@ -160,4 +160,23 @@
         </select>
     </div>
 </div>
+
+{{!-- Prompt Configuration --}}
+<div class="form-group">
+    {{#if (eq item.type "consumable")}}
+    <label class="checkbox">
+        <input type="checkbox" name="system.uses.autoDestroy" {{checked system.uses.autoDestroy}}> {{ localize "DND5E.ItemDestroyEmpty" }}
+    </label>
+    {{/if}}
+    {{#if item.hasAreaTarget}}
+    <label class="checkbox">
+        <input type="checkbox" name="system.target.prompt" {{checked system.target.prompt}}> {{ localize "DND5E.TemplatePrompt" }}
```

Comment:
Suggestion for tooltip: "If unchecked, the prompt for placing a Measured Template will be suppressed."